monitoring/README.md

```diff
@@ -72,10 +93,11 @@ This example shows how to deploy monitoring tools (Prometheus, Grafana and Pyros
 
     a) WDS API Server: input into the following text boxes.
```

Comment:
This dashboard is misnamed. It is not only for WDS(es), it is also for ITS(es) and the KubeFlex hosting cluster.

Reply:
Fixed!